packages/super-components/components/msc-billing-tile/src/msc-billing-tile/msc-billing-tile.tsx

```diff
@@ -0,0 +1,219 @@
+import {
+  Component,
+  Prop,
+  h,
+  Element,
+  Host,
+  State,
+  Fragment,
+} from '@stencil/core';
+import {
+  OdsThemeColorIntent,
+  OdsThemeTypographyLevel,
+  OdsThemeTypographySize,
+} from '@ovhcloud/ods-theming';
+import { HTMLStencilElement, Watch } from '@stencil/core/internal';
+import { Locale, defaultLocale, formatDate } from '@ovhcloud/msc-utils';
+import { apiClient } from '@ovh-ux/manager-core-api';
+import { getTranslations } from './translations';
+import {
+  ServiceDetails,
+  ServiceInfos,
+  Translations,
+} from './msc-billing.types';
+
+export interface IMscBillingTile {
+  servicePath: string;
+  locale?: Locale;
+  commitmentDataTracking?: string;
+  changeOwnerDataTracking?: string;
+  updateOwnerDataTracking?: string;
+  subscriptionManagementDataTracking?: string;
+  renewLinkDataTracking?: string;
+  cancelResiliationDataTracking?: string;
+  manageRenewDataTracking?: string;
+  resiliateDataTracking?: string;
+  anticipateRenewDataTracking?: string;
+}
+
+@Component({
+  tag: 'msc-billing-tile',
+  styleUrl: 'msc-billing-tile.scss',
+  shadow: true,
+})
+export class MscBillingTile implements IMscBillingTile {
+  @Element() host!: HTMLStencilElement;
+
+  @Prop() public locale = defaultLocale;
+
+  @Prop() public servicePath: string;
+
+  @Prop() public commitmentDataTracking?: string = '';
+
+  @Prop() public changeOwnerDataTracking?: string;
+
+  @Prop() public updateOwnerDataTracking?: string;
+
+  @Prop() public subscriptionManagementDataTracking?: string;
+
+  @Prop() public renewLinkDataTracking?: string;
+
+  @Prop() public cancelResiliationDataTracking?: string;
+
+  @Prop() public manageRenewDataTracking?: string;
+
+  @Prop() public resiliateDataTracking?: string;
+
+  @Prop() public anticipateRenewDataTracking?: string;
+
+  @Prop() public changeOfferDataTracking?: string;
+
+  @State() localeStrings?: Translations;
+
+  @State() serviceInfos?: ServiceInfos;
+
+  @State() serviceDetails?: ServiceDetails;
+
+  @Watch('locale')
+  async updateTranslations() {
+    this.localeStrings = await getTranslations(this.locale);
+  }
+
+  async componentWillLoad() {
+    this.updateTranslations();
+
+    apiClient.v6.get(`${this.servicePath}/serviceInfos`).then((response) => {
+      this.serviceInfos = response.data;
+      apiClient.v6
+        .get(`/services/${this.serviceInfos?.serviceId}`)
+        .then((responseDetails) => {
+          this.serviceDetails = responseDetails.data;
+        });
```

Comment:
```suggestion
        .then((response) => {
          this.serviceDetails = response.data;
        });
```

Reply:
`response` variable name is already used in the upper scope
Shadowing the variable creates an eslint warning